.travis.yml

```diff
@@ -22,13 +31,13 @@ before_script:
 
   # Install composer dependencies
   - composer validate
-  - if [[ $DB == PGSQL ]]; then composer require --no-update silverstripe/postgresql 2.0.x-dev; fi
-  - composer install --prefer-dist --no-interaction --no-progress --no-suggest --optimize-autoloader --verbose --profile
+  - if [[ $DB == PGSQL ]]; then composer require --no-update silverstripe/postgresql:^2; fi
+  - composer install --prefer-source --no-interaction --no-progress --no-suggest --optimize-autoloader --verbose --profile
 
 script:
   - if [[ $PHPUNIT_TEST ]]; then vendor/bin/phpunit; fi
   - if [[ $PHPUNIT_COVERAGE_TEST ]]; then phpdbg -qrr vendor/bin/phpunit --coverage-clover=coverage.xml; fi
-  - if [[ $PHPCS_TEST ]]; then vendor/bin/phpcs src/ tests/; fi
+  - if [[ $PHPCS_TEST ]]; then vendor/bin/phpcs --standard=vendor/silverstripe/framework/phpcs.xml.dist src/ tests/; fi
```

Comment:
Sorry I didn't get a chance to review all of your PRs. Each module has its own ruleset. One of the reasons for that is that we can stick much more closely to PSR-12 this way, where framework has a tonne of exceptions that we shouldn't enforce on all of the modules. The other reason is because (as you've changed above) we require the dist files to optimise build time.

Reply:
good point, thanks!

Reply:
fixed